### PR TITLE
Reindent everything with ocp-indent

### PR DESCRIPTION
**Pull request by robhoes**
Adds a reindent makefile target to help doing this.

Signed-off-by: Rob Hoes <rob.hoes@citrix.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -122,3 +122,6 @@ report:
 	bisect-ppx-report -I _build -html $(COVERAGE) /tmp/bisect-xenops*out
 
 .PHONY: report
+
+reindent:
+	ocp-indent --inplace **/*.ml*
```
Comment (mseri) [inline Makefile]: Please use 
```
git ls-files '*.ml' '*.mli' | xargs ocp-indent --syntax cstruct -i
```
from https://github.com/xapi-project/xcp-idl/blob/master/Makefile

Comment (edwintorok) [reply Makefile]: Actually do we use the `cstruct` syntax extension anywhere? I only added it because I've seen it on one of the ring3 training examples.

Comment (mseri) [reply Makefile]: I am not sure, we were already using it somewhere. I copied it from one of our repositories when I made the example.

**Comment by krizex** [comment]
This is a long-awaited pull request. ;-)

**Comment by robhoes** [comment]
@krizex I knew that you would like it :)

**Pull request merged**